apps/web/src/components/dashboard/dashnav.tsx

```diff
@@ -1,43 +1,55 @@
 "use client";
 
+import { useQuery } from "@tanstack/react-query";
 import { useSession } from "next-auth/react";
 import Link from "next/link";
 import { redirect } from "next/navigation";
 import { usePathname } from "next/navigation";
 
-import organizationsMock from "~/lib/mock/organizations";
 import { cn } from "~/lib/utils";
+import { getOwnOrganizations } from "~/server/actions/organization";
 
 export default function Dashnav() {
-  const organizations = organizationsMock;
+  const { data: organizations } = useQuery({
+    queryKey: ["organizations"],
+    queryFn: async () => {
+      const [organizations, err] = await getOwnOrganizations();
+
+      if (err !== null) {
+        console.error(err);
+      }
+
+      return organizations;
+    },
+  });
 
   const pathname = usePathname();
   const session = useSession();
-  const orgId = pathname.split("/").at(2);
+  const orgName = pathname.split("/").at(2);
 
   if (session.status === "unauthenticated") {
     redirect("/");
   }
 
   const links = [
     {
-      path: `/dashboard/${orgId}/overview`,
+      path: `/dashboard/${orgName}/overview`,
       label: "overview",
     },
     {
-      path: `/dashboard/${orgId}/settings`,
+      path: `/dashboard/${orgName}/settings/general`,
```

Comment:
We don't have any other subpage on settings. Change `general` to root page on `/settings/`

Reply:
done